### ISSUE TITLE
Vardefs definition and custom data in DOM

### ISSUE THREAD

**Issue by Abuelodelanada** [salesagility/SuiteCRM#3006]
When developing some complex JS functionality is very useful to have module vardefs definitions or custom data in the DOM so you can read it from JS.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Vardefs definition and custom data in DOM](https://trello.com/c/ikthNYNG/35-vardefs-definition-and-custom-data-in-dom)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
